topics/admin/tutorials/connect-to-compute-cluster/slides.html

```diff
@@ -124,7 +124,7 @@
 For Java tools, be sure to set `-Xmx`, e.g.:
 
 ```xml
-<destination id="foo" ...>
+<Destination id="foo" ...>
```

Comment:
This doesn't seem correct.

Reply:
```suggestion
<destination id="foo" ...>
```

feel free to use suggestions so I can just click on them since your suggestions are always right :)